mail_tracking/static/src/components/failed_message/failed_message.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+    <t
+        t-name="mail_tracking.FailedMessage"
+        t-inherit="mail.Message"
+        t-inherit-mode="primary"
+    >
+        <!-- Avoid setting original message as isActive upon hovering failed message-->
+        <xpath expr="//div[hasclass('o_Message')]" position="attributes">
+            <attribute
+                t-if="this.props.isFailedMessage"
+                name="t-on-mouseenter"
+            >messageView.onMouseenterFailed</attribute>
+        </xpath>
+        <xpath expr="//p[hasclass('o_mail_tracking')]" position="replace">
```

Comment:
Avoid replaces

Reply:
@luisg123v the use of replace was intentional here because I'm using `t-inherit-mode="primary"` to generate a new component altogether, so the future inheritances of `mail.Message` will not be affected by this. Adding the `d-none ` class instead would result in the original recipients header to be shown when inspecting the element, which I personally dislike since the invisible elements are still rendered and do not contribute to the purpose of the component.

Knowing this, I believe there is no danger in using replace here, but what do you think?

Reply:
OK.

BTW, the correct way to ide is using `t-if="False"`, not hiding it using classes.

Reply:
I see, thanks!